homeassistant/components/soundtouch/media_player.py

```diff
@@ -78,36 +78,35 @@ class SoundTouchMediaPlayer(MediaPlayerEntity):
     _attr_device_class = MediaPlayerDeviceClass.SPEAKER
     _attr_has_entity_name = True
     _attr_name = None
+    _attr_source_list = [
+        Source.AUX.value,
+        Source.BLUETOOTH.value,
+    ]
 
     def __init__(self, device: SoundTouchDevice) -> None:
         """Create SoundTouch media player entity."""
 
-        self._device = device
+        self.device = device
 
-        self._attr_unique_id = self._device.config.device_id
+        self._attr_unique_id = device.config.device_id
         self._attr_device_info = DeviceInfo(
-            identifiers={(DOMAIN, self._device.config.device_id)},
+            identifiers={(DOMAIN, device.config.device_id)},
             connections={
-                (CONNECTION_NETWORK_MAC, format_mac(self._device.config.mac_address))
+                (CONNECTION_NETWORK_MAC, format_mac(device.config.mac_address))
             },
             manufacturer="Bose Corporation",
-            model=self._device.config.type,
-            name=self._device.config.name,
+            model=device.config.type,
+            name=device.config.name,
         )
 
         self._status = None
         self._volume = None
         self._zone = None
 
-    @property
-    def device(self):
-        """Return SoundTouch device."""
-        return self._device
```

Comment:
Just out of interest: what is the reason for removing the property here?

Reply:
Honestly, I didn't really see the difference between having it as a private property `_device` and exposing it through a property and just having it as a property.

I think this looks cleaner as it's now just 1 variable that holds the device (not 2, `_device` and `device`). Also when I look in the code, only the `_device` one is used, so I did not really see the point of exposing it like this.

But this is just my opinion, maybe you have a different reason and I would love to know why :)

Reply:
You are right, the `device` property doesn't seem to be used anywhere outside this class (anymore) as far as I can see.
If so, I would suggest leaving the field as `_device` to indicate it is private.

Reply:
That's fine with me